mindmeld/constants.py

```diff
@@ -12,4 +12,6 @@
 # limitations under the License.
 DEFAULT_TRAIN_SET_REGEX = r"train.*\.txt"
 DEFAULT_TEST_SET_REGEX = r"test.*\.txt"
-DEVCENTER_URL = "https://devcenter.mindmeld.com"
+DEVCENTER_URL = "https://blueprints.mindmeld.com"
```

Comment:
It's probably best to rename this, since "devcenter" isn't a good description of what we're using it for anymore.

Reply:
done